bfcfg

```diff
@@ -53,21 +54,24 @@ mlxmkcap=/lib/firmware/mellanox/boot/capsule/scripts/mlx-mkcap
 
 bfcf_acpi_table=/sys/firmware/acpi/tables/BFCF
 efivars=/sys/firmware/efi/efivars
-sys_cfg_sysfs=${efivars}/BfSysCfg-9c759c02-e5a3-45e4-acfc-c34a500628a6
-redfish_cfg_sysfs=${efivars}/BfRedfish-ce3e5882-6770-4d5e-aa45-90f909da2446
-redfish_state_cfg_sysfs=${efivars}/BfRedfishState-ce3e5882-6770-4d5e-aa45-90f909da2446
-sb_state_sysfs=${efivars}/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c
-sb_setup_mode_sysfs=${efivars}/SetupMode-8be4df61-93ca-11d2-aa0d-00e098032b8c
 efi_global_var_guid=8be4df61-93ca-11d2-aa0d-00e098032b8c
 efi_bfcfg_var_guid=487ff588-fb71-4b19-9100-ebe067aa1af0
 efi_vlan_var_guid=9e23d768-d2f3-4366-9fc3-3a7aba864374
+efi_redfish_var_guid=ce3e5882-6770-4d5e-aa45-90f909da2446
+efi_debug_var_guid=f3ce977e-c17f-493e-a3cd-5731d386e505
+sys_cfg_sysfs=${efivars}/BfSysCfg-9c759c02-e5a3-45e4-acfc-c34a500628a6
+redfish_cfg_sysfs=${efivars}/BfRedfish-${efi_redfish_var_guid}
+redfish_state_cfg_sysfs=${efivars}/BfRedfishState-${efi_redfish_var_guid}
+sb_state_sysfs=${efivars}/SecureBoot-${efi_global_var_guid}
+sb_setup_mode_sysfs=${efivars}/SetupMode-${efi_global_var_guid}
 rshim_efi_mac_sysfs=${efivars}/RshimMacAddr-${efi_global_var_guid}
 pxe_dhcp_class_id_sysfs=${efivars}/DhcpClassId-${efi_global_var_guid}
 bf_bundle_version_sysfs=${efivars}/BfBundleVer-${efi_bfcfg_var_guid}
 sb_custom_mode_sysfs=${efivars}/BfCfgSbCustomMode-${efi_bfcfg_var_guid}
 pass_settings_sysfs=${efivars}/BfCfgPassSettings-${efi_bfcfg_var_guid}
 bf_modes_sysfs=${efivars}/BfCfgBfModes-${efi_bfcfg_var_guid}
 boot_wdog_sysfs=${efivars}/BfBootWdog-120b4e59-904a-4fe2-9e03-54e9a29ffb22
+osinfo_sysfs=${efivars}/OsInfo-${efi_debug_var_guid}
```

Comment:
Q: why you chose the variable name `OsInfo` versus `BfOsInfo`?

The UEFI spec defines standard variables, for platform specific variables it is recommended to use a platform specific prefix so it won't conflict with future specifications

Reply:
Done, updated.